### PR TITLE
Lab1

### PR DESCRIPTION
**Pull request by TheKatarGit**
Lab 1 Functional

### CODE REVIEW

**Comment by TheKatarGit** [comment]
@ed-asriyan

File: C Programming/Lab 1/InOut.h [hunk 1]
```diff
@@ -1,23 +1,22 @@
 #include "stdio.h"
 #include "stdlib.h"
 
-int read_from_file(float ArrayOfNumbers[])
+void read_from_file(float ArrayOfNumbers[], int *NumberOfElements)
```
Comment (TheKatarGit) [inline C Programming/Lab 1/InOut.h:4]: @ed-asriyan Why this works then?

Comment (ed-asriyan) [reply C Programming/Lab 1/InOut.h:4]: What was the mistake prior to this commit?

Comment (TheKatarGit) [reply C Programming/Lab 1/InOut.h:4]: This is about Pointers and other mess from Telegram chat. Skip it if you don't remember

**Comment by ed-asriyan** [comment]
lgfm

**Pull request merged**